app/views/organisers/index.html.slim

```diff
@@ -5,6 +5,13 @@
       p
       p
       p
+      = form_tag organisers_path, :method => :post do
```

Comment:
because our form contains only one input we can use flash for that so in this else https://github.com/LearnITGirl/WebPlatform/pull/14/files#diff-11b31c89a012a8032400be786ecb15e5R15 instead of assigning "User exist" to flash please do this `@user.errors.full_messages.join(", ")` - that will shows us all errors